src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt

```diff
@@ -194,7 +194,10 @@ class Cell(
       addSpecialistCellType(it, userOrSystemInContext, clock)
     }
 
-    if (accommodationType == AccommodationType.NORMAL_ACCOMMODATION) {
+    if (accommodationType == AccommodationType.NORMAL_ACCOMMODATION ||
```

Comment:
I might have checked that it's not OTHER rather than checking each of the other options, but it achieves the same thing. This is perhaps just a little more computationally expensive.

Reply:
... and harder to read

Reply:
@danbenton-mojdt right but I don't know exactly what Business want to do with OTHER... 
A) is a Non Res and it can be a CELL? and capacity is 0. but it might have? 
B) it should not be a CELL ?
I have created a ticket for OTHER if at least we can discuss that with the Business . by default i would leave in this way.

Reply:
can just use `in`

Reply:
I added OTHER as placeholder for cells I didn't know what to do with as part of migration.  We need to not allow OTHER and definitely now allow this API call to set it in this context.  I think the Enum will have to change to only allow the 3 other types.

Reply:
@Mjwillis  We can do 'Enum will have to change to only allow the 3 other types.'under MAP-1445' for now I commit what agreed in MAP-1438 (this PR)

Reply:
@Mjwillis  I have API call not to pass OTHER however i can adress Enum later which is simpler.. ..